README.md

```diff
@@ -48,11 +48,19 @@ var childWalletSecret = lib.derivePublic(parentWalletPublicKey, 1, 1)
 * `Buffer toPublic(Buffer privateKey)`
 * `Buffer decodePaperWalletMnemonic(string paperWalletMnemonic)`
 * `Buffer xpubToHdPassphrase(Buffer xpub)`
-* `string packAddress(Array[int] derivationPath, Buffer xpub, Buffer hdPassphrase, int derivationScheme)`
+* `Buffer packBootstrapAddress(Array[int] derivationPath, Buffer xpub, Buffer hdPassphrase, int derivationScheme)`
```

Comment:
this should pack also the protocol magic in case of testnet byron adresses, see https://docs.cardano.org/projects/adrestia/en/latest/key-concepts/addresses-byron.html#example-2-daedalus-address-byron-testnet